### PR TITLE
add ignore config

### LINKED ISSUE #1: test，我
test1111

### LINKED ISSUE #8: issue8了
还好了

### PR DESCRIPTION
**Pull request by jomatto**
config about ignore

### CODE REVIEW

**Comment by jomatto** [comment]
rrrr

**Comment by Jeffmen** [comment]
lock

**Comment by Jeffmen** [comment]
vvvv

**Review by Jeffmen — COMMENTED** [review]
test

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/profile/build
+/library/plugin/build
```
Comment (Jeffmen) [inline .gitignore:2]: commet

Comment (Jeffmen) [reply .gitignore:2]: 很好

**Review by Jeffmen — DISMISSED** [review]
edit

**Comment by Jeffmen** [comment]
locked

**Review by Daojiand — COMMENTED** [review]
comment from daojiand

**Pull request closed without merging**